virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc

```diff
@@ -0,0 +1,26 @@
+[id="virt-configuring-pci-passthrough"]
+= Configuring PCI passthrough
+include::modules/virt-document-attributes.adoc[]
+:context: virt-configuring-pci-passthrough
+
+toc::[]
+
+//This assembly contains the content for
+//configuring PCI passthrough by using the CLI. There are
+//plans to enable PCI passthrough configuration
+//by using the web console (next release).
+//When this feature is available in the web console, please
+//add the new content to this assembly.
+
+include::modules/virt-about-pci-passthrough.adoc[leveloffset=+1]
+include::modules/virt-preparing-nodes-and-cluster-pci.adoc[leveloffset=+1]
+include::modules/virt-adding-kernel-arguments-enable-iommu.adoc[leveloffset=+2]
+include::modules/virt-binding-devices-vfio-driver.adoc[leveloffset=+2]
+include::modules/virt-adding-pci-device-hyperconverged-operator.adoc[leveloffset=+2]
+include::modules/virt-configuring-vms-for-pci-passthrough.adoc[leveloffset=+1]
+include::modules/virt-assigning-pci-device-virtual-machine.adoc[leveloffset=+2]
+
+== Additional resources
+* For information about enabling virtualization hardware extensions, refer to link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/virtualization_deployment_and_administration_guide/sect-troubleshooting-enabling_intel_vt_x_and_amd_v_virtualization_hardware_extensions_in_bios[Enabling Intel VT-X and AMD-V Virtualization Hardware Extensions in BIOS].
+* For information about file permissions, refer to  link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_basic_system_settings/assembly_managing-file-permissions_configuring-basic-system-settings[Managing file permissions].
+* For information about the Machine Config Operator, refer to link:https://github.com/openshift/machine-config-operator#applying-configuration-changes-to-the-cluster[machine-config-operator]
```

Comment:
@ousleyp I verified with the RHEL team - the links shown here are for the official RHEL documentation.  Vendula and Jiri Herrmann in the RHEL doc team confirmed that these are official docs for RHEL. Therefore,  we are keeping these links because the details provided there are relevant and useful for users who are configuring PCI passthrough.

Reply:
Thanks for making sure!